src/main/resources/wa-task-configuration-civil-civil.dmn

```diff
@@ -147,7 +147,7 @@ else "1"</text>
           <text>-</text>
         </inputEntry>
         <inputEntry id="UnaryTests_0nmr8951">
-          <text>"reviewSpecificAccessRequestJudiciary","reviewSpecificAccessRequestAdmin","reviewSpecificAccessRequestLegalOps"</text>
+          <text>"reviewSpecificAccessRequestJudiciary","reviewSpecificAccessRequestAdmin","reviewSpecificAccessRequestLegalOps", "reviewSpecificAccessRequestsCTSC"</text>
```

Comment:
Typo in reviewSpecificAccessRequestsCTSC fix it to reviewSpecificAccessRequestCTSC